apollo-router/src/services/supergraph_service.rs

```diff
@@ -184,7 +187,7 @@ where
             if can_be_deferred && !accepts_multipart(req.originating_request.headers()) {
                 let mut response = SupergraphResponse::new_from_graphql_response(graphql::Response::builder()
                     .errors(vec![crate::error::Error::builder()
-                        .message(String::from("the router received a query with the @defer directive but the client does not accept multipart/mixed HTTP responses"))
+                        .message(String::from("the router received a query with the @defer directive but the client does not accept multipart/mixed HTTP responses. To enable @defer support, add the HTTP header Accept: multipart/mixed; deferSpec=20220824"))
```

Comment:
Not sure why but it's a bit hard for me to parse the suggestion here, but that's probably because of the semi-colon in the middle.  I'm not sure if my single-quoting makes it any easier, but just a thought

```suggestion
                        .message(String::from("the router received a query with the @defer directive but the client does not accept multipart/mixed HTTP responses. To enable @defer support, add the HTTP header 'Accept: multipart/mixed; deferSpec=20220824'"))
```

We can fix this later though.